Patches/Core/ThingDefs_Races/Races_Mechanoid.xml

```diff
@@ -449,4 +449,19 @@
 		</value>
 	</Operation>
 
+	<!--Only apply if Biotech isn't loaded-->
```

Comment:
Gotta patch the ammo in its loadout and the Advanced Pikeman from VFE-MEchanoids too.

Reply:
Pikemen are already limited by the significant bulk values of the shell, conditionally patching that won't change anything gameplay-wise while creating an inefficient rat's nest of nested patch ops

Reply:
Pikeman mechs are set to carry 9~10 mags worth of shells being 27~30 total shells, leaving the loadout as it will max their inventory capacities out...

Reply:
Changing the mag spawns to the old ones wouldn't change anything in game, the mech would still spawns with only 9 shells, which is max capacity it can hold, but it'll take several conditional FindMods to do, doing so in an exercise in futility

I would also really appreciate if you weren't behaving like a total asshole

Reply:
So you think that maxing out bulk or mass capacity of the mech is a good decision? 
Fair enough I guess.

Reply:
Mate, that's literally how it used to be, old loadout spawns still maxed out their capacity, the amount of shells spawned on them won't change